edb/edb11.Dockerfile

```
@@ -1,4 +1,4 @@
-FROM quay.io/enterprisedb/edb-postgres-advanced:10.22
+FROM quay.io/enterprisedb/edb-postgres-advanced:11.20-3.3-postgis
 USER root
 RUN chown -R postgres:postgres /var/lib/edb && chmod 777 /var/lib/edb && mkdir /docker-entrypoint-initdb.d
```

I think you'll need this.

```suggestion
RUN chown -R postgres:postgres /var/lib/edb && chmod 777 /var/lib/edb && rm /docker-entrypoint-initdb.d/10_postgis.sh
```

See the commented out lines below and compare that with the edb14.Dockerfile.